coalib/output/ConsoleInteraction.py

```diff
@@ -597,7 +597,12 @@ def acquire_settings(log_printer, settings_names_dict, section):
     for setting_name, arr in sorted(settings_names_dict.items(),
                                     key=lambda x: (join_names(x[1][1:]), x[0])):
         value = require_setting(setting_name, arr, section)
-        result.update({setting_name: value} if value is not None else {})
+
+        while str(value).strip() == '':
+            print(str(setting_name) + ' setting left blank.')
```

Comment:
Wouldn't it make the message clearer if it had ` setting is required.` instead of ` setting left blank.` ? This would stress upon the fact that code won't proceed unless the user provides a value for the setting. 

But then again, I am not sure if such a message is the norm for linux CLI tools.

Reply:
Yes, you are right. 
I'm open for suggestions about the warning message that should be printed because I wasn't really sure what the appropriate message would be in this situation.